services/galley/src/Galley/API/Internal.hs

```diff
@@ -112,6 +113,11 @@ internalAPI =
       <@> iTeamsAPI
       <@> mkNamedAPI @"upsert-one2one" iUpsertOne2OneConversation
       <@> featureAPI
+      <@> federationAPI
+
+federationAPI :: API IFederationAPI GalleyEffects
+federationAPI =
+  mkNamedAPI @"get-federation-status" getFederationStatus
```

Comment:
We only need this when creating a conversation and when we add users to a conversation (so far). Because this is in galley, we can directly call the handler. If it was in brig, we would have to do a remote call to brig, which sucks. Also, please note that the internal endpoint only exists to make this testable. What we actually need in production is the code from the handler.